src/org/zaproxy/zap/extension/pscan/PolicyPassiveScanPanel.java

```diff
@@ -237,4 +249,56 @@ public void setPassiveScanTableModel(PolicyPassiveScanTableModel categoryTableMo
     public String getHelpIndex() {
         return "ui.dialogs.options.pscanrules";
     }
+
+	private static JPanel createFilterPanel(final JTable table) {

```

Comment:
Might worth extracting a top level class (e.g. `TableFilterPanel`) from this method?